docker-vertica/packages/cleanup.sh

```diff
@@ -34,11 +34,17 @@ rm -r -f \
    /opt/vertica/examples \
    /opt/vertica/sdk \
    /opt/vertica/packages/*/examples \
-   /opt/vertica/oss/python3/lib/python3.7/test \
-   /opt/vertica/oss/python3/lib/python3.7/pip \
-   /opt/vertica/oss/python3/lib/python3.7/site-packages/pip \
-   /opt/vertica/oss/python3/lib/python3.7/config-3.7*
+   /opt/vertica/packages/kafka \
+   /opt/vertica/oss/python3/lib/python3.9/test \
+   /opt/vertica/oss/python3/lib/python3.9/unittest/test \
+   /opt/vertica/oss/python3/lib/python3.9/pip \
+   /opt/vertica/oss/python3/lib/python3.9/site-packages/pip \
+   /opt/vertica/oss/python3/lib/python3.9/config-3.9* \
+   /opt/vertica/oss/python3/lib/python3.9/tkinter \
+   /opt/vertica/oss/python3/lib/python3.9/idlelib 
 
+# cleanup many of the __pycache__ directories 
+find /opt/vertica/oss/python3/lib/python3.9/  -type d -name "__pycache__" -exec rm -rf {} +
```

Comment:
There are __pycache__ directories off of /opt/vertica/oss/python3/bin.  Can we use /opt/vertica/oss as the directory?  That way we don't have to update this if the python version changes again.